### PR TITLE
[feature]: hide complete test summary in case of app not running

### PR DESCRIPTION
**Pull request by AhmedLotfy02**
## Related Issue
  - Hide complete test summary in case of app not running

Closes: #1577

#### Describe the changes you've made
-If there are testcases then the test summary and coverage opportunity will appear otherwise not.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added
- I tested it with and without testcases.
Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by AhmedLotfy02** [comment]
I wanted to make a make the test summary runs at the end of the **Test** function but the error of permissions won't be caught there as it's in cmd/test.go so can i make a function in utils that prints the test summary at the end of cmd/test.go
another solution is to make the file existence check inside service/test/test.go at the end of the function and use defer to print the test summary at the end of **Test** function

File: pkg/service/test/test.go [hunk 1]
```diff
@@ -300,36 +300,39 @@ func (t *tester) Test(path string, testReportPath string, generateTestReport boo
...
+			// merges the coverage files into a single txt file which can be merged with the go-test coverage
+			generateCovTxtCmd := exec.Command("go", "tool", "covdata", "textfmt", "-i="+os.Getenv("GOCOVERDIR"), "-o="+os.Getenv("GOCOVERDIR")+"/total-coverage.txt")
+			output, err = generateCovTxtCmd.Output()
+			if err != nil {
+				t.logger.Error("failed to get the coverage of the go binary", zap.Error(err), zap.Any("cmd", coverCmd.String()))
+			}
+			if len(output) > 0 {
+				t.logger.Sugar().Infoln("\n", models.HighlightFailingString(string(output)))
+			}
 		}
```
Comment (shivamsouravjha) [inline pkg/service/test/test.go:333]: this print doesn't seem to give a good UX  better to remove it rest looks fine to me

Comment (AhmedLotfy02) [reply pkg/service/test/test.go:333]: @shivamsouravjha Done

Comment (AhmedLotfy02) [reply pkg/service/test/test.go:333]: I think you mean the print in the else right?
I removed it.

Comment (shivamsouravjha) [reply pkg/service/test/test.go:333]: why did you changed the totaltests to < 0? The complete test summary should be displayed after total sets is more than 0

Comment (AhmedLotfy02) [reply pkg/service/test/test.go:333]: it's mistake will i was testing , I returned it.

**Review by shivamsouravjha — APPROVED** [review]
LGTM!

**Pull request merged**